package.json

```diff
@@ -36,6 +37,7 @@
   },
   "devDependencies": {
     "chai": "^3.2.0",
+    "compression": "^1.6.0",
```

Comment:
I think you need to modify your test to install dependencies in the scaffolded application.

```js
var install = require('strong-cached-install');
var PKG_CACHE = path.resolve(__dirname, '..', '.pkgcache');
// ...

beforeEach(function installDependencies(done) {
  install(SANDBOX, PKG_CACHE, ['dependencies'], done);
});
```